ocs_ci/deployment/factory.py

```diff
@@ -6,6 +6,7 @@
 from .azure import AZUREIPI
 from .vmware import VSPHEREUPI
 from .baremetal import BAREMETALUPI
+from .azure import AZUREIPI
```

Comment:
This line seems like a remnant from the rebase, it could be safely dropped as it's the same as line 6 above.

Reply:
fixed !